docs/guides/extension_points.rst

```diff
@@ -83,11 +83,11 @@ If you wish to customize aspects of the learner or educator experiences, you'll
 
 Most python plugins are enabled using one of two methods:
 
-1. A Python Entry point: the core Open edX platform provides a standard plugin loading mechanism in |openedx.core.lib.plugins|_ which uses `stevedore`_ to find all installed python packages that declare a specific "entry point" in their setup.py file. See the ``entry_points`` defined in edx-platform's own ``setup.py`` for examples.
+1. A Python Entry point: the core Open edX platform provides a standard plugin loading mechanism in |edx_django_utils.plugins.plugin_manager|_ which uses `stevedore`_ to find all installed python packages that declare a specific "entry point" in their setup.py file. See the ``entry_points`` defined in edx-platform's own ``setup.py`` for examples.
```

Comment:
1. Cache and Monitoring expose public API via `__init__.py` module in edx-django-utils. Should we do the same here rather than adding plugin_manager here?
2. If you think it is important to highlight the plugin manager here, should we mention PluginManager?

Reply:
We may want to discuss this one later.